RedHat-UX/red-hat-design-system
# docs: vpn links

Closes #592 

## What I did

1. add styles for `a.vpn`: displaying the brand padlock icon after vpn links
2. add `vpn` class to vpn links, including ctas on fts page


## Testing Instructions

1. check any element's code page installation section
2. check fts page

## Notes to Reviewers
This is a straw PR to get the ball rolling. if design likes it then great we can merge, otherwise we'll iterate until it satisfies.
